sql_generators/glean_usage/__init__.py

```diff
@@ -34,11 +35,74 @@
     clients_last_seen_joined.ClientsLastSeenJoined(),
 ]
 
-# * mlhackweek_search was an experiment that we don't want to generate tables
-# for
-# * regrets_reporter currently refers to two applications, skip the glean
+# regrets_reporter currently refers to two applications, skip the glean
 # one to avoid confusion: https://github.com/mozilla/bigquery-etl/issues/2499
-SKIP_APPS = ["mlhackweek_search", "regrets_reporter", "regrets_reporter_ucs"]
+SKIP_APPS = ["regrets_reporter", "regrets_reporter_ucs"]
+APP_LISTINGS_URL = "https://probeinfo.telemetry.mozilla.org/v2/glean/app-listings"
+
+
+def get_app_info():
```

Comment:
I moved these 4 functions from `common.py` to here, since they aren't used anywhere else. The idea is that we should move functions as close to where they get used as possible.